gui/src/vmm/hw/console/context.rs

```diff
@@ -89,7 +85,7 @@ enum ExecError {
     InvalidLen,
 
     #[error("invalid message")]
-    InvalidMsg,
+    InvalidMsg(#[from] std::string::FromUtf8Error),
```

Comment:
This can cause wrong behavior if someone accidentally `String::from_utf8(v)?` on a non-msg field.

Reply:
Yeah, that was copilot. :sweat_smile: